ios/RNZoomUs.m

```diff
@@ -239,4 +291,65 @@ - (void)onMeetingError:(MobileRTCMeetError)errorCode message:(NSString *)message
   meetingPromiseReject = nil;
 }
 
+- (NSArray<NSString *> *)supportedEvents
+{
+  return @[@"InMeetingEvent"];
+}
+
+- (void) notifyInMeetingEvent:(NSString *)event params:(NSDictionary *)params {
+    NSDictionary *body = @{
+        @"event": event,
+        @"payload": params
+    };
+    [self sendEventWithName:@"InMeetingEvent" body:body];
+}
+
+- (NSDictionary *)getUSerInfoByUserId:(NSUInteger)userID {
+    MobileRTCMeetingService *ms = [[MobileRTC sharedRTC] getMeetingService];
+    MobileRTCMeetingUserInfo *userInfo = [ms userInfoByID:userID];
+    return @{
+        @"userId": [NSString stringWithFormat:@"%li",  userID],
+        @"name": userInfo.userName
+        // @"participantId": userInfo.participantID
+    };
+}
+
+- (void)onSinkMeetingActiveVideo:(NSUInteger)userID {
+    [self notifyInMeetingEvent:@"meeting.user.video.active" params:[self getUSerInfoByUserId:userID]];
+}
+
+- (void)onSinkMeetingVideoStatusChange:(NSUInteger)userID {
+    MobileRTCMeetingService *ms = [[MobileRTC sharedRTC] getMeetingService];
+    MobileRTCMeetingUserInfo *userInfo = [ms userInfoByID:userID];
+    BOOL active = [[userInfo videoStatus] isSending];
+    [self notifyInMeetingEvent:@"meeting.user.video.status" params:@{
+        @"userId": [NSString stringWithFormat:@"%li",  userID],
+        @"name": userInfo.userName,
+        @"active": [NSNumber numberWithBool:active]
+    }];
+}
+
+- (void)onSinkMeetingAudioStatusChange:(NSUInteger)userID {
+    MobileRTCMeetingService *ms = [[MobileRTC sharedRTC] getMeetingService];
+    MobileRTCMeetingUserInfo *userInfo = [ms userInfoByID:userID];
+    BOOL muted = [[userInfo audioStatus] isMuted];
+    [self notifyInMeetingEvent:@"meeting.user.audio.status" params:@{
+        @"userId": [NSString stringWithFormat:@"%li",  userID],
+        @"name": userInfo.userName,
+        @"muted": [NSNumber numberWithBool:muted]
+    }];
```

Comment:
You are outputing different params per different events.
If it is possible, I think it would make it cleaner to always output the same param per any event